client/src/components/J40Map.module.scss

```diff
@@ -1,36 +1,75 @@
 @use '../styles/design-system.scss' as *;
 @import "./utils.scss";
 
-.j40Popup {
-  width: 375px;
-}
 
-.navigationControl {
-  left: .75em;
-  top: units(15);
-  // width: 2.5em;
-}
+.j40Map {
+  @include at-media-max("desktop") {
+    // < 1024
+    height: 55vh;
+  }
 
-.fullscreenControl {
-  right: 1.25em;
-  top: 2.5em;
-}
+  .mapHeaderRow{
+    @include u-display("flex");
+    @include u-padding-left(1.5);
+    
+    justify-content: space-between;
```

Comment:
I cannot find `justify-content: space-between` in the [caniuse site](https://caniuse.com/?search=justify-content). Can you verify it's supported by all your target browsers?

Reply:
Great catch! Works on Chrome 103 , FF 102 and Safari 15.5